docs/docs/versions.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Versions
+layout: docs
+---
+
+Because the TSC library is a client for the Tableau Server REST API, you need to confirm that the version of the TSC
+library that you use is compatible with the version of the REST API used by your installation of Tableau Server.
+
+* TOC
+{:toc}
+
+## Display the REST API version
+
+To display the default version of the REST API used by TSC, run the following code:
+
+```py
+import tableauserverclient as TSC
+
+server = TSC.Server('http://SERVER_URL')
+
+print(server.version)
```

Comment:
It's always 2.3 by default, unless you change it to 2.4.

We don't auto detect or anything like that, so running code to discover this feels unnecessary